lib/collection/response.js

```diff
@@ -259,6 +259,25 @@ _.assign(Response, /** @lends Response */ {
     _postman_propertyName: 'Response',
 
     /**
+     * Converts the response object from the request module to the postman responseBody format
+     *
+     * @param {Object} response The response object, as received from the request module
+     * @returns {Object} The transformed responseBody
+     * @todo Add a key: `originalRequest` to the returned object as well, referring to response.request
+     */
+    fromModule: function (response, cookies) {
```

Comment:
The function name has to be `.createFromRequest` since `fromModule` is super vague.